lib/r10k/git/errors.rb

```diff
@@ -10,24 +10,24 @@ class UnresolvableRefError < GitError
       attr_reader :ref
       attr_reader :git_dir
 
-      def initialize(*args)
+      def initialize(mesg, options = {})
+        if mesg.is_a? Hash
```

Comment:
What's the reason for receiving options through both an options hash and `mesg` as options hash?